pinot-core/src/main/java/com/linkedin/pinot/core/operator/transform/function/DateTimeConversionTransform.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.core.operator.transform.function;
+
+import com.google.common.base.Preconditions;
+import com.linkedin.pinot.common.data.DateTimeFieldSpec;
+import com.linkedin.pinot.common.data.FieldSpec;
+import com.linkedin.pinot.common.utils.time.DateTimeFieldSpecUtils;
+import com.linkedin.pinot.core.common.BlockValSet;
+import com.linkedin.pinot.core.plan.DocIdSetPlanNode;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+
+/**
+ * This class implements the date time conversion transform.
+ *
+ * <ul>
+ *   <li> This udf should be invoked with arguments: </li>
+ *   <ul>
+ *     <li> 1. Column name to convert eg. Date </li>
+ *     <li> 2. format(as defined in {@link DateTimeFieldSpec}) of the input column. eg: 1:HOURS:EPOCH, 1:DAYS:SIMPLE_DATE_FORMAT:yyyyMMdd </li>
+ *     <li> 3. format(as defined in {@link DateTimeFieldSpec}) of the output expected. eg: 1:MILLISECONDS:EPOCH, 1:WEEKS:EPOCH, 1:DAYS:SIMPLE_DATE_FORMAT:yyyyMMdd </li>
```

Comment:
This fails style check, ie long than 160 characters.

Reply:
Done